plugins/building-hacks.cpp

```diff
@@ -168,7 +180,8 @@ struct work_hook : df::building_workshopst{
     }
     DEFINE_VMETHOD_INTERPOSE(void, categorize, (bool free))
```

Comment:
can this get run on world load before scripts have a chance to register custom workshop types?

Reply:
Ugh this is a difficult issue... Lua script enabling this workshop might not have run yet as they usually run on world load.